face_autoencoder/image_processing.py

```diff
@@ -21,3 +23,37 @@ def __call__(self, image: np.ndarray) -> torch.Tensor:
 
     def preprocess_image(self, image: np.ndarray) -> torch.Tensor:
         return self.image_preprocessing_pipeline(Image.fromarray(image))
+
+
+class PreprocessingPipelineWithEdges:
+    def __init__(self, nn_input_image_resolution: int, weights: int) -> None:
+        self.nn_input_image_resolution = nn_input_image_resolution
+        self.weights = weights
+        self.image_preprocessing_pipeline = transforms.Compose(
+            [
+                transforms.ToTensor(),
+                transforms.Resize(
+                    (nn_input_image_resolution, nn_input_image_resolution)
+                ),
+            ]
+        )
+
+    def __call__(
+        self, image: np.ndarray, edge: np.ndarray
+    ) -> t.Tuple[torch.Tensor, torch.Tensor]:
+        return self.preprocess_image(image, edge)
+
+    def preprocess_image(
```

Comment:
Rozdziel to na 2 funkcje jedna do przetwarzania obrazow (preprocess_image) a 2 do preprocessowania wag (preprocess weights)